### DISCUSSION TITLE
Ability to move (drag & drop) an item in one timeline to another timeline

### DISCUSSION THREAD

**Discussion by sshearer** [dialogic-godot/dialogic#646] [Ideas]
Could you please consider adding the ability for me as a content creator to select 1 or more items with 1 timeline and move the selection into another timeline ?

I see the ability exists to move items around WITHIN a timeline.  I'm asking to extend it further.

Thanks,
Steven.

**Comment by Jowan-Spooner** [comment]
You can select some events hit CTRL+X then go to another timeline and paste them with CTRL+V. I don't think that drag and drop will be an easy thing to implement.
